README.md

```diff
@@ -1,8 +1,10 @@
 # MQTT Client Library
 
-This repository contains an ISO C90 compliant MQTT client library designed for embedded platforms.
+This repository contains the coreMQTT Client library that has been optimized for a low memory footprint.  The coreMQTT Client library implements the complete [MQTT 3.1.1](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/mqtt-v3.1.1.html) standard.  It has no dependencies on any additional libraries other than the standard C library and a customer-implemented network transport interface. This library is distributed under the [MIT License](LICENSE).
```

Comment:
This repository contains the coreMQTT client library that has been optimized for low memory footprint embedded devices, and implements the MQTT 3.1.1 standard. It has no dependencies on any additional libraries other than the standard C library and a customer-implemented network transport interface. This library is distributed under the MIT open source license.

Reply:
another Q: what is 'customer-implemented'?

Reply:
Could we instead use "_user_-implemented" or "application-developer" instead of "customer" to make it more relevant for the reader to understand?